apps/avatax/src/modules/avatax/calculate-taxes/avatax-calculate-taxes-payload-lines-transformer.ts

```diff
@@ -33,11 +50,27 @@ export class AvataxCalculateTaxesPayloadLinesTransformer {
     });
 
     if (taxBase.shippingPrice.amount !== 0) {
+      const shippingAmountMinusDiscounts = new Decimal(taxBase.shippingPrice.amount)
+        .sub(new Decimal(this.calculateShippingDiscount(taxBase.discounts)))
+        .toNumber();
+
+      if (shippingAmountMinusDiscounts < 0) {
+        Sentry.captureException(
+          new Error("Saleor returned shipping discounts with higher values than shipping amount"),
```

Comment:
I dont see a reason to. 

Modern Error is great if we return/throw error, because we can match it by instance later. 
Here error is locally thrown only for Sentry, and constructor is needed only to capture stack.

Not sure if there is any benefit of throwing subclass of error here?

Reply:
ohh, ok I see 👍🏻